### ISSUE TITLE
getProjectTasks() returning undefined

### ISSUE THREAD

**Issue by g33k5z** [7eggs/node-toggl-api#2]
Hi. 

I have been using your npm package for over a month without issues before this. So first up thank you.

**My Issue**:
```javascript
getProjectTasks(projectId, callback)
````
I am unable to get this function to return task data. It only seems to return undefined. 

Again all other functions work as expected so **I'm inclined to believe this is a mistake on my end**, but I don't see where.

**Steps taken so far**:

* Remade code:
```javascript
 router.get('/getProjectTasks/:id', function(req, res) {
        toggl.getProjectTasks(req.params.id, function(err, tasks){ 
              res.json(tasks); 
            });
    });
````

* Updated npm package

* Verified I can get data directly via 
https://www.toggl.com/api/v8/projects/8{projectId}/tasks?api_token={my_token}
and also with cURL

* Current workaround: 
Using ````getWorkspaceTasks(wId, options, callback)```` and then flittering tasks per project as needed

**Comment by estliberitas** [comment]
Hello! You're right. Fixing.

**Comment by estliberitas** [comment]
Published `0.0.3`. You can do now:

    npm install toggl-api@0.0.3 --save

**Comment by g33k5z** [comment]
Thank you!
